bes_theme/assets/data/OSSP-master.js

```diff
@@ -0,0 +1,178 @@
+[
+    {
+        "id": 50,
+        "github_id": 391939220,
```

Comment:
refer #53 
There is no "github_id" , this js is agnostic of github. 


 "id": xx,                                                            //same as tracking_id 
    "bes_tracking_id": xx,

Reply:
@panickervinod Just wanted the id of the project (from https://api.github.com/orgs/Be-Secure/repos) to be there so that it is easier to identify the project.

Reply:
@asa1997 do you mean id of all github repos associated to the project ?  then it should be under project repos.

Reply:
@panickervinod Ok Vinod. Thought it would be easier to pick it up if I have it outside any other key.